images/ubuntu/hub/Dockerfile

```diff
@@ -16,8 +16,11 @@ RUN sh -c 'echo "deb https://hub.unity3d.com/linux/repos/deb stable main" > /etc
  && apt-get -q install -y "unityhub=$hubVersion" \
  && apt-get clean
 
+# Bug in xvfb-run that causes it to redirect stderr to stdout. We want it separate
```

Comment:
Do you have details about that bug?

Reply:
https://bugs.launchpad.net/ubuntu/+source/xorg-server/+bug/1059947
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=868876